packages/analytics/amplify_analytics_pinpoint_dart/pubspec.yaml

```diff
@@ -9,20 +9,20 @@ environment:
   sdk: ^3.3.0
 
 dependencies:
-  amplify_core: ">=2.3.0 <2.4.0"
+  amplify_core: ">=2.4.0 <2.5.0"
   amplify_db_common_dart: ">=0.4.4 <0.5.0"
   amplify_secure_storage_dart: ">=0.5.1 <0.6.0"
```

Comment:
Min should be bumped: 
`amplify_db_common_dart: ">=0.4.5 <0.5.0"`
`amplify_secure_storage_dart: ">=0.5.2 <0.6.0"`

Reply:
Updated!